api/src/main/java/org/openmrs/api/db/hibernate/HibernateOrderDAO.java

```diff
@@ -52,50 +54,49 @@
 import java.util.Locale;
 import java.util.Set;
 
-
 /**
  * This class should not be used directly. This is just a common implementation of the OrderDAO that
  * is used by the OrderService. This class is injected by spring into the desired OrderService
  * class. This injection is determined by the xml mappings and elements in the spring application
  * context: /metadata/api/spring/applicationContext.xml.<br>
  * <br>
  * The OrderService should be used for all Order related database manipulation.
- * 
+ *
  * @see org.openmrs.api.OrderService
  * @see org.openmrs.api.db.OrderDAO
  */
 public class HibernateOrderDAO implements OrderDAO {
-	
+
```

Comment:
There are so many formatting changes in this file. Do you mind discarding them?

Reply:
@miirochristopher did you see the above?

Reply:
I tried to fix the formatting with the diff tool but seems to have no effect.

Reply:
Yes, tests are underway, then I will push. Thanks, @dkayiwa.